webpack.config.js

```diff
@@ -33,6 +33,17 @@ module.exports = {
         test: /\.css$/i,
         use: ['style-loader', 'css-loader'],
       },
+      {
+        test: /\.(png|jp(e*)g|svg|gif)$/,
+        use: [
+          {
+            loader: 'file-loader',
```

Comment:
While this works file-loader was [deprecated](https://v4.webpack.js.org/loaders/file-loader/) in webpack v5. [Asset Modules](https://v4.webpack.js.org/loaders/file-loader/) are the new way to do this

Reply:
Added a rule for asset/resource as an alternative to file-loader. Have not uninstalled file-loader. The docs were showing examples of inline asset management for svg, so I don't know if that would be a better module to implement.